tests/chainer_tests/training_tests/extensions_tests/test_evaluator.py

```diff
@@ -42,11 +42,18 @@ def __init__(self, return_values):
         self.iterator = iter(return_values)
         self.finalized = False
         self.return_values = return_values
+        self.current_position = 0
```

Comment:
Those attributes (`current_position`, `epoch_size` and `epoch_detail`) are not requirements in a minimal iterator. Please undo them.

Reply:
Thanks. Fixed.